docs/explanations/fix_scenarios.rst

```diff
@@ -56,8 +57,10 @@ CVE/USN without a released fix
 ==============================
 
 Some CVEs/USNs do not have a fix released yet. When that happens, ``pro fix``
-will let you know! Before we reproduce this scenario, let us first install a
-package that we know has no fix available by running:
+will let you know! This is example output created in the past, there
```

Comment:
On the shared `pro-fix-simple-case`, we still have this sentence here:
`Therefore let us install an older package on the VM`

That  doesn't fit well in this new doc. However, since we mentioned we could reuse the VM setup from the tutorial right now, maybe this is fine.

Reply:
Yeah, this isn't too wrong to keep nor too hard to rewrite. Let us wait until @s-makin can join as a tie breaker and then we can merge it in either style.

Reply:
It's only in one place that I can see in the rendered page, and since you've now got the preamble suggesting following the tutorial to get the VM, I don't think it's completely out of place. Happy to leave it as is for now!